src/src/main/java/org/redfrog404/mobycraft/entity/EntityChaosMonkey.java

```diff
@@ -40,16 +38,14 @@
 	 */
 	private int maxDistance = 25;
 
-	MobycraftCommandsFactory factory = MobycraftCommandsFactory.getInstance();
```

Comment:
This is still around -- look lower in the file -- but as a more tightly scoped local variable rather than a class-scoped one.